Automattic/wp-calypso
# Help Center: Connect to zendesk  when user wants to contact support

Related to pdDR7T-1Vr-p2#comment-2342

## Proposed Changes

Create a connection to Zendesk when users click on "Contact our Support Team"

<img width="421" alt="image" src="https://github.com/user-attachments/assets/07881f61-3f87-4f4c-8fcf-f34d748cca77">

I made in a way to avoid adding "Feeling stuck" or feedback to the message switching you to zendesk

## Why are these changes being made?


We still had to update that link

## Testing Instructions



* Add the flag `?flags=help-center-experience`
* Open Wapuu
* Ask something
* Click on "Contact our support team"
* You should be talking with a HE (fake if you're in staging)
